foundry/applications/formApplications/imagePopout.d.ts

```diff
@@ -86,6 +86,10 @@ declare class ImagePopout extends FormApplication<ImagePopout.Data, string> {
 
 declare namespace ImagePopout {
   interface Options extends FormApplication.Options {
+    /**
+     *  @defaultValue `''`
+     */
+    title: string;
```

Comment:
I don't think we need to add this, it is inherited from `Application.Options`.

Reply:
Yeah this should be done differently. We should remove all of the duplicated explicit types in the `Options` type here and only add what was added for this class. Then document the default values as one thing on the `get defaultOptions` getter.